module-services/service-db/include/service-db/QuotesMessages.hpp

```diff
@@ -387,6 +387,37 @@ namespace Quotes
             }
         };
 
+        class ReadRandomizedQuoteRequest : public db::Query
+        {
+          public:
+            explicit ReadRandomizedQuoteRequest() : Query(Query::Type::Read)
```

Comment:
explicit for the default constructor (no args) has no effect

Reply:
true, leftover after copying

Reply:
removed